docs/spec/spec.md

```diff
@@ -452,3 +385,32 @@ status:
 
   observedGeneration: ...  # last generation being reconciled
 ```
+
+## Container
+
+This is a [core.v1.Container](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#container-v1-core).
+Some fields are not allowed, such  as name, ports, and volumeMounts.
```

Comment:
nit: extra space `such  as`.  We also just allowed `ports` through cc @dgerd

Reply:
Fixed, also added documentation for container ports.